features.json

```diff
@@ -141,6 +167,119 @@
               }
             ]
           }
+        },
+        {
+          "key": "unpaywallEmailAddressKey",
+          "type": "input",
+          "templateOptions": {
```

Comment:
Since no `defaultValue` was provided, this means it'll be an empty string, or perhaps even null or undefined, correct?

Reply:
Should be `undefined` from what I can tell at-least:
https://github.com/formly-js/angular-formly/blob/c1bd043dc56bed4eecb573717c60804e993e1ab0/src/directives/formly-field.js#L193

https://github.com/formly-js/angular-formly/blob/c1bd043dc56bed4eecb573717c60804e993e1ab0/src/directives/formly-field.js#L130

They use a project called Angular Formly:
http://angular-formly.com/#!/